assets/settings/default.json

```diff
@@ -71,6 +71,12 @@
   // The factor to grow the active pane by. Defaults to 1.0
   // which gives the same size as all other panes.
   "active_pane_magnification": 1.0,
+  // Active pane styling settings.
+  "active_pane_modifiers": {
+    "magnification": 1.0,
```

Comment:
Let's add some brief descriptions with allowed value ranges here too, to keep in line with the rest of the json.

Reply:
and I should remove the old value from the `default.json` I assume, correct?

Reply:
Yes, same as from the *.md file.

Reply:
done